test/functional/test-cid.js

```diff
@@ -596,8 +613,7 @@ describe('cid', () => {
         cookieName: 'cookie_name',
       }, hasConsent).then(c => {
         expect(c).to.exist;
-        expect(c).to
-            .equal('amp-sha384([1,2,3,0,0,0,0,0,0,0,0,0,0,0,0,15])');
+        expect(c).to.equal('amp-AQIDAAAAAAAAAAAAAAAADw');
```

Comment:
Yes, at the moment it really is a full 384 bits, packed into a full 64 chars at 6bits/char.